util/compression.cc

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
Since this is a new file, we can say 2022-present. And LevelDB copyright is not necessary.